cmd/resetHeaderCheckCount.go

```diff
@@ -34,7 +34,8 @@ var (
 var resetHeaderCheckCountCmd = &cobra.Command{
 	Use:   "resetHeaderCheckCount",
 	Short: "Resets header check_count for the given block number",
-	Long: fmt.Sprintf(`Resets check_count to zero for the given header so that the execute command may recheck that header's logs in case one was missed.
+	Long: fmt.Sprintf(`Run this command to reset the check_count to zero for a given header so that
```

Comment:
Not introduced by you but I'm wondering if we might want to be consistent about whether to use `fmt.Sprintf` to interpolate the flag names. I think I would lean toward yes, we should be doing it - so that we can update the flag name without updating the docs. But in that case I'd probably also use it for e.g. `starting-block-number` in `headerSync`